.github/workflows/acceptance-tests.yml

```diff
@@ -47,7 +47,6 @@ jobs:
           generic:
             - 'mongodbatlas/data_source_mongodbatlas_backup_compliance_policy*.go'
             - 'mongodbatlas/resource_mongodbatlas_backup_compliance_policy*.go'
-            - 'mongodbatlas/data_source_mongodbatlas_data_lakes_test*.go'
             - 'mongodbatlas/resource_mongodbatlas_x509_authentication_database_user*.go'
             - 'mongodbatlas/resource_mongodbatlas_auditing*.go'
```

Comment:
Could you also make sure to delete the documentation and examples related to these resources?

Reply:
Good point! Let me do that.

Reply:
@andreaangiolillo done